<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="seedValueSpinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="newGamePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Uusi peli</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberSec">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="setFlagCheckBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Set flag</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberSize">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
